src/com/intellij/idea/plugin/hybris/settings/HybrisProjectImportApplicationSettingsConfigurableProvider.kt

```diff
@@ -100,6 +102,21 @@ class HybrisProjectImportApplicationSettingsConfigurableProvider : ConfigurableP
                             .bindText(state::groupCCv2)
                     }.layout(RowLayout.PARENT_GRID)
                 }.visibleIf(groupModulesCheckBox.selected)
+                row {
+                    externalModulesCheckBox = checkBox("Group external modules")
+                        .bindSelected(state::groupExternalModules)
+                        .component
+
+                }
+                indent {
+                    row {
+                        icon(HybrisIcons.MODULE_EXTERNAL_GROUP)
```

Comment:
add a comment stating what modules will be grouped (Eclipse, Maven, Gradle)

Reply:
![image](https://github.com/epam/sap-commerce-intellij-idea-plugin/assets/7203327/e699bca3-6214-4943-8d7e-81add9907df5)
done